encode/h264encode.c

```diff
@@ -1088,17 +1104,29 @@ static int init_va(void)
         h264_profile = profile_list[i];
         vaQueryConfigEntrypoints(va_dpy, h264_profile, entrypoints, &num_entrypoints);
         for (slice_entrypoint = 0; slice_entrypoint < num_entrypoints; slice_entrypoint++) {
-            if (entrypoints[slice_entrypoint] == VAEntrypointEncSlice) {
+            if (requested_entrypoint == -1 ){
```

Comment:
Could you please use a space between  ')' and '{' ?

Reply:
done